.github/workflows/runtimeCheck.yml

```diff
@@ -12,22 +14,16 @@ jobs:
   Check-For-Changes-To-Runtime-File:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: tj-actions/changed-files@v35
-        id: changed_files
-      - name: Check if runtime.ts has been modified
-        id: detect_runtime_change
-        run: |
-          runtime_changed='false'
-          for changed_file in ${{ steps.changed_files.outputs.all_changed_files }}; do
-            if [ "$changed_file" = "packages/teams-js/src/public/runtime.ts" ]; then
-              runtime_changed='true'
-            fi
-          done
-          echo "runtime_changed=$runtime_changed" >> $GITHUB_OUTPUT
-      - name: Comment on PR
-        if: steps.detect_runtime_change.outputs.runtime_changed == 'true'
-        uses: mb2dev/github-action-comment-pull-request@1.0.0
+      - name: 'Comment on PR'
+        if: github.event.pull_request.head.repo.fork != true
+        uses: actions/github-script@v6
         with:
-          message: "This pull request contains changes to the runtime.ts file. If you, as the author of this PR, have made changes to the Runtime interface please review RUNTIME.md to determine if a new runtime version is required. Please reply to this comment stating what changes, if any, were made to the runtime object and whether a new runtime version was required."
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            let fs = require('fs');
```

Comment:
is the let `fs = require('fs')` needed? I assume it's for the following github command, but just wanted to make sure.

Additionally, since we no longer allow for builds on forked repositories, I believe your next steps section isn't needed.

Reply:
Hmm, I think this is left over from the old implementation. I'll take it out, and good callout on the next steps, I'll update the description